### PR TITLE
Add parser to the python api

### PR DESCRIPTION
**Pull request by daniel-larraz**

### CODE REVIEW

File: src/api/python/cvc5.pxd [hunk 1]
```diff
@@ -559,3 +570,27 @@ cdef extern from "<cvc5/cvc5.h>" namespace "cvc5":
...
+        Command() except +
+        void invoke(Solver* solver, SymbolManager* sm, ostream& out) except +
+        string toString() except +
+        string getCommandName() except +
+        bint isNull() except +
+
+    cdef cppclass InputParser:
+        InputParser(Solver* solver, SymbolManager* sm) except +
+        void setFileInput(InputLanguage lang, const string& filename) except +
+        #void setStreamInput(InputLanguage lang, istream& input, const string& name) except +
```
Comment (ajreynol) [inline src/api/python/cvc5.pxd:591]: You can delete this.

**Review by ajreynol — CHANGES_REQUESTED** [review]
Can you also add the python equivalent of the examples: 
https://github.com/cvc5/cvc5/blob/main/examples/api/cpp/parser.cpp
https://github.com/cvc5/cvc5/blob/main/examples/api/cpp/parser_sym_manager.cpp
to the directory https://github.com/cvc5/cvc5/tree/main/examples/api/python?

@mudathirmahgoub this also is missing for Java.

File: src/api/python/cvc5.pxd (hunk 1 above)
Comment (daniel-larraz) [reply src/api/python/cvc5.pxd]: Done

**Comment by daniel-larraz** [comment]
> Can you also add the python equivalent of the examples: https://github.com/cvc5/cvc5/blob/main/examples/api/cpp/parser.cpp https://github.com/cvc5/cvc5/blob/main/examples/api/cpp/parser_sym_manager.cpp to the directory https://github.com/cvc5/cvc5/tree/main/examples/api/python?
> 

Added

**Review by ajreynol — APPROVED** [review]

**Pull request merged**